package.json

```diff
@@ -1,25 +1,29 @@
 {
-  "name": "peerplays-electron-template",
-  "version": "0.0.1",
-  "description": "The intent of this project is to provide a starting point for building electron applications that connect to the Peerplays BlockChain.",
+  "name": "peerplays-electron-build-bookie",
+  "version": "1.0.0",
```

Comment:
some changes needed here. This is still saying the name is from the last repo.
Two branches exist so we need this package.json sync to be in both those branches without their names/versions/descriptions being modified.